bot/exts/info/information.py

```diff
@@ -36,47 +29,60 @@ def __init__(self, bot: Bot):
         self.bot = bot
 
     @staticmethod
-    def role_can_read(channel: GuildChannel, role: Role) -> bool:
-        """Return True if `role` can read messages in `channel`."""
-        overwrites = channel.overwrites_for(role)
-        return overwrites.read_messages is True
-
-    def get_staff_channel_count(self, guild: Guild) -> int:
-        """
-        Get the number of channels that are staff-only.
+    def is_staff_channel(guild: Guild, channel: GuildChannel) -> bool:
```

Comment:
This can become a utility function under utils/channel
#1355 will benefit from it.

Reply:
Resolved in ca38700.